cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,13 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-    
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link ').click();
+    cy.get('[placeholder="Email"]').type('irina@test.qa');
+    cy.get('[placeholder="Password"]').type('qwerty');
+    cy.get('.btn').click();
+
+    cy.get(':nth-child(4) > .nav-link').should('have.text', 'testuseririna');
```

Comment:
Add a test case for checking if the 'Sign in' page is visited successfully.

Reply:
Done